app/src/main/res/values/strings.xml

```diff
@@ -28,6 +28,7 @@
   <string name="enter_password_activity_unlock_button">Unlock Wallet</string>
   <string name="enter_password_activity_creating_wallet_progress_dialog">Creating Wallet…</string>
   <string name="enter_password_activity_unlocking_wallet_progress_dialog">Unlocking Wallet…</string>
+  <string name="enter_password_activity_xml_universe">Connecting to universe&#160;</string>
```

Comment:
If possible, I would recommend we try to paste the actual unicode characters rather than the corresponding HTML codes. If not possible, a short descriptive comment could be helpful.

Reply:
hmmm, I need to look into this. This is meant to be a space character as I concatenate the universe and change the colour of the text when dealing with this string

Reply:
https://developer.android.com/guide/topics/resources/string-resource#formatting-strings